tests/unittest/_torch/attention/sparse/test_sparse_attention.py

```
@@ -0,0 +1,1022 @@
+
+
+def convert_sparse_attn_ctx_indices_to_global(
+    sparse_attn_ctx_indices: torch.Tensor,
+    metadata: TrtllmAttentionMetadata,
+    layer_idx: int = 0,
+    kv_factor: int = 2,
+) -> torch.Tensor:
+    """
+    Convert local sparse_attn_ctx_indices to global KV cache pool indices.
+    """
+    num_kv_heads, num_tokens, num_sparse_tokens = sparse_attn_ctx_indices.shape
```

Ruff (0.14.13)

Unpacked variable `num_tokens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)